src/autopas/selectors/tuningStrategy/ActiveHarmony.h

```diff
@@ -232,9 +238,10 @@ void ActiveHarmony::fetchConfiguration() {
   } else {
     cellSizeFactor = ah_get_real(htask, cellSizeFactorsName);
   }
+  int verletRebuildFrequency = _allowedVerletRebuildFrequencies->getMin();
```

Comment:
This should be handled like above for `cellSizeFactor`

Reply:
Alternatively, if it shouldn't be handled like `cellSizeFactor`, add a comment to explain why.